tests/functional/test_libraries.py

```diff
@@ -8,15 +8,17 @@
 #-----------------------------------------------------------------------------
 
 
+# Library imports
+# ---------------
 import os
 import pytest
+import shutil
 
+# Local imports
+# -------------
 from PyInstaller.compat import is_win
 from PyInstaller.utils.tests import importorskip, xfail_py2
-
-
-# Directory with data for some tests.
-_DATA_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data')
+from tests.functional.conftest import DATA_DIR
```

Comment:
This line is erroring because `tests/` is no longer a Python module (`__init__.py` was removed).

To avoid importing anything from `conftest.py` in the test suites, I suggest making DATA_DIR into a fixture that will be passed into the test function as an argument. See the functions marked `@pytest.fixture` at the end of `conftest.py` for examples.

Another option is to move the `DATA_DIR` global into `PyInstaller.utils.tests` instead - but this would be tricky as the actual definition is in `tests/functional/scripts`, so a fixture is probably the way to go.